bin/gce-delete-instances

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
Please merge this script with `gce-delete-instance`.  Two scripts differing in the trailing character may cause confusion.

`gce-delete-instace --all` may work?